app/main.py

```diff
@@ -3,13 +3,14 @@
 from app.api.v1.api import api_router
 from app.mcp.server import mcp
 
-mcp_app = mcp.http_app(path="/")
+# Note: Using path="/" causes connection issues with Claude
+mcp_app = mcp.http_app(path="/mcp")
 
 app = FastAPI(lifespan=mcp_app.lifespan)
 
 app.include_router(api_router, prefix="/api/v1")
 
-app.mount("/mcp", mcp_app)
+app.mount("/sparkth-mcp", mcp_app)
```

Comment:
Let's move the MCP endpoints to /ai/mcp and document the public endpoint in the README.